index.js

```diff
@@ -4,6 +4,14 @@ var async = require('async')
 var MongoClient = require('mongodb').MongoClient
 
 function clean (db, done) {
+  cleanDb(db, null, done)
+}
+
+function cleanExclude (db, exclude, done) {
+  cleanDb(db, exclude, done)
+}
+
```

Comment:
I thought about an options parameter too, but i was wondering about the compatibility with the standard clean(db/url, cb) call. Maybe a check on the type of the second parameter could be a good solution? If it's a function behave like the current version, otherwise (if it's an object) check the parameters?